### ISSUE TITLE
Unable to create credentials

### ISSUE THREAD

**Issue by tnkemdilim** [bitwarden/sdk#33]
### Steps To Reproduce

I keep getting a `Resource not found` whenever I attempt to make a request to create a credential. 

```rust
let request = SecretCreateRequest {
    organization_id: "...",
    key: "some_key".to_owned(),
    value: "some value".to_owned(),
    note: "".to_string(),
};

self.client.secrets().create(&request).await?;
```

One significant thing I noticed is this [line](https://github.com/bitwarden/sdk/blob/fad3c019765ede86dea867219dd4d5af98f576ee/crates/bitwarden/src/commands/secrets.rs#L47) which is always provided as `None` instead of expecting the project id from the caller.

### Expected Result

I should be able to able to create credentials.

### Actual Result

```json
{
    "message": "Resource not found.",
    "validationErrors": null,
    "exceptionMessage": null,
    "exceptionStackTrace": null,
    "innerExceptionMessage": null,
    "object": "error"
}
```

### Operating System

macOS

### Build Version

0.2.1

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Hinton** [comment]
Hi @tnkemdilim,

Service Accounts & Access tokens are currently read only and can't be used create secrets or projects which is mentioned in the docs https://bitwarden.com/help/access-tokens/. I know the team is exploring adding support for this in the future though.

**Comment by tnkemdilim** [comment]
Thanks for the response @Hinton 

Can the API key be used instead? One thing that wasn't clear to me though was the need to provide a master password when using the API key approach. Should the master key be required?

**Comment by tnkemdilim** [comment]
Thanks for the response @Hinton

A few more questions:
- Is there an alternative? I could use the API key approach, but it doesn't provide the option of restricting access to a project (correct me if I got this wrong).
- Is there a tentative date for when this will be available?

**Comment by Hinton** [comment]
This should be resolved in version 0.3.0 which was released a few days ago. https://bitwarden.com/help/secrets-manager-cli/#secret-create
